internal/website/content/howto/runtimevar/_index.md

```diff
@@ -1,53 +1,91 @@
-function in the driver package directly (like `awsparamstore.OpenVariable`).
-This guide shows how to use both forms for each provider.
+The easiest way to do so is to use [`runtimevar.OpenVariable`][] and a URL pointing
+to the variable, making sure you ["blank import"][] the driver package to link
+it in. See [Concepts: URLs][] for more details. If you need fine-grained control
+over the connection settings, you can call the constructor function in the
+driver package directly (like `etcdvar.OpenVariable`).
 
-[Concepts: URLs]: {{< ref "/concepts/urls.md" >}}
+When opening the variable, you can provide a [decoder][] parameter (either as a
+query parameter for URLs, or explicitly to the constructor) to specify whether
+the raw value stored in the variable is interpreted as a `string`, a `[]byte`,
+or as JSON.
+
+See the [guide below][] for usage of both forms for each supported provider.
+
+[`*runtimevar.Variable`]: https://godoc.org/gocloud.dev/runtimevar#Variable
+[`runtimevar.OpenVariable`]: https://godoc.org/gocloud.dev/runtimevar#OpenVariable
 ["blank import"]: https://golang.org/doc/effective_go.html#blank_import
-[`runtimevar.OpenVariable`]:
-https://godoc.org/gocloud.dev/runtimevar#OpenVariable
+[Concepts: URLs]: {{< ref "/concepts/urls.md" >}}
+[decoder]: https://godoc.org/gocloud.dev/runtimevar#Decoder
+[guide below]: {{< ref "#services" >}}
+
+## Using a Variable {#using}
+
+Once you have opened a `runtimevar.Variable` for the provider you want, you can
+use it portably.
 
-## GCP Runtime Configurator {#gcp-url}
+### Latest {#latest}
+
+The easiest way to a `Variable` is to use the [`Variable.Latest`][] method. It
+returns the latest good [`Snapshot`][] of the variable value, blocking if no
+good value has *ever* been received. The dynamic type of `Snapshot.Value`
+depends on the decoder you provided when creating the `Variable`.
+
+To avoid blocking, you can pass an already-`Done` context.
+
+{{< goexample src="gocloud.dev/runtimevar.ExampleVariable_Latest"
+imports="0" >}}
+
+[`Variable.Latest`]: https://godoc.org/gocloud.dev/runtimevar#Variable.Latest
+[`Snapshot`]: https://godoc.org/gocloud.dev/runtimevar#Snapshot
+
+### Watch {#watch}
+
+`Variable` also has a [`Watch`][] method for obtaining the value of a variable;
+it has different semantics than `Latest` and may be useful in some scenarios. We
+recommend starting with `Latest` as it's conceptually simpler to work with.
+
+[`Watch`]: https://godoc.org/gocloud.dev/runtimevar#Variable.Watch
+
+## Supported Services {#services}
+
+### GCP Runtime Configurator {#gcprc}
 
 To open a variable stored in [GCP Runtime Configurator][] via a URL, you can use
 the `runtimevar.OpenVariable` function as follows.
 
+[GCP Runtime Configurator]: https://cloud.google.com/deployment-manager/runtime-configurator/
+
 {{< goexample
 "gocloud.dev/runtimevar/gcpruntimeconfig.Example_openVariableFromURL" >}}
 
-### GCP Runtime Configurator Constructor {#rc-ctor}
+#### Constructor {#gcprc-ctor}
```

Comment:
Maybe the compromise would be GCP Constructor, AWS Constructor?

Reply:
OK, done.